pyannote/audio/interactive/common/utils.py

```diff
@@ -107,15 +106,22 @@ def remove_audio_before_db(examples: List[Dict]) -> List[Dict]:
     return examples
 
 
-def get_chunks(source: Path, chunk_duration: Optional[float] = None):
-
-    # Diff and review recipes can give only one file as source
-    if os.path.isdir(source):
+def source_to_files(source: Path) -> List[Dict]:
+    """
+    Convert a directory or a file path to a list of files object for prodigy
+    """
+    if source.is_dir():
         files = ProdigyAudioLoader(source)
     else:
-        name = os.path.basename(source).rsplit(".", 1)[0]
+        name = source.name.rsplit(".", 1)[0]
```

Comment:
Did you mean to extract the stem? 
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem

Reply:
Yes precisely... Thanks